### PR TITLE
stan: new port in math, C++ package for Bayesian inference

### PR DESCRIPTION
**Pull request by barracuda156**
#### Description

New port in math: https://github.com/stan-dev/stan
Second component for `Stan` installation. Once this is merged, I have everything for `stanc3`, Stan OCaml interpreter.

###### Type(s)

- [ ] bugfix
- [ ] enhancement
- [ ] security fix

###### Tested on

macOS 10.6.8 Server
Xcode 3.2.6

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ ] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL? 
- [x] checked your Portfile with `port lint --nitpick`?
- [ ] tried existing tests with `sudo port test`?
- [ ] tried a full install with `sudo port -vst install`?
- [ ] tested basic functionality of all binary files?
- [ ] checked that the Portfile's most important [variants](https://trac.macports.org/wiki/Variants) haven't been broken?

### CODE REVIEW

**Comment by barracuda156** [comment]
@herbygillot May I ask you to take a look? This is a dependent of `stanmath`.

File: math/stan/Portfile [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+PortSystem              1.0
+PortGroup               github 1.0
+
+github.setup            stan-dev stan 2.31.0 v
+categories              math
+maintainers             {@barracuda156 gmail.com:vital.had} openmaintainer
+license                 BSD
+description             C++ package for Bayesian inference, maximum likelihood estimation etc.
+long_description        ${description}
```
Comment (herbygillot) [inline math/stan/Portfile:11]: `${description}` needs to be expanded here: `{*}${description}`

Comment (barracuda156) [reply math/stan/Portfile:11]: Oops, sorry again, I had this port for quite a while, missed to update this part. Will do now.

**Comment by herbygillot** [comment]
Looks good @barracuda156, thanks.

**Pull request merged**